packages/vuepress/vuepress-plugin-apidocs/lib/metadata/processor.js

```diff
@@ -99,9 +93,10 @@ class MetadataProcessor {
         memberMetadata.returns.summary = this.renderMarkdown(memberMetadata.returns.summary)
       }
 
-      if (memberType === 'properties' && this.constantNamingPattern.test(memberMetadata.name)) {
-        this.hasConstants = true
-        return
+      if (memberType === 'properties') {
+        if (memberMetadata.constants) {
```

Comment:
So from my local testing, the JSON seems to have the already expanded array of constant strings (it seems to do the resolution of wildcards to the full array). So this code here (the `resolveConstants` call and this guard) is unnecessary and actually breaks it (because it assumes a `string` value and not a `string[]`